src/display/optional_content_config.js

```diff
@@ -65,10 +65,46 @@ class OptionalContentConfig {
       }
       return this._groups.get(group.id).visible;
     } else if (group.type === "OCMD") {
-      // Per the spec, the expression should be preferred if available. Until
-      // we implement this, just fallback to using the group policy for now.
+      // Per the spec, the expression should be preferred if available.
       if (group.expression) {
-        warn("Visibility expression not supported yet.");
+        const evaluateExpression = array => {
```

Comment:
Can we please move this helper function to a "private" method, rather than defining it inline?